src/main/java/com/google/devtools/build/lib/worker/WorkerKey.java

```diff
@@ -68,6 +71,8 @@
     this.mustBeSandboxed = mustBeSandboxed;
     /** Set it to true if this job should be run with WorkerProxy. */
     this.proxied = proxied;
+    /** If multiplex worker is used, it should be "multiplex-worker", otherwise return "worker" */
```

Comment:
This comment doesn't make a lot of sense here. Where do these magic strings come from? If they are important, they should be part of the docstring. And if they must be these specific values, there should be some validation. I would remove this.